entrypoint.sh

```diff
@@ -12,12 +12,12 @@ else
 fi
 
 # Define a common command prefix
-WORKER_CMD="celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=$CONCURRENCY"
+WORKER_CMD="celery -A run_celery.notify_celery worker --logfile=/dev/null --concurrency=$CONCURRENCY"
```

Comment:
I guess no point in having the loglevel if it is all going to /dev/null, right?

Reply:
Indeed